portfolio/src/main/webapp/animations.js

```diff
@@ -0,0 +1,46 @@
+/* intro section animations */
+
+gsap.to('.layer-1', { y: '-100vh', delay: .5 });
+gsap.to('.layer-2', { y: '-100vh', delay: .7 });
+gsap.to('.layer-3', { y: '-100vh', delay: .9 });
+gsap.to('#overlay', { y: '-100vh', delay: 1.5 });
+gsap.to('.circle', 20, { rotation:"360", ease: Linear.easeNone, repeat: -1 });
+
+gsap.fromTo('.circle-small', 
+  { x: '10rem', opacity: 0 }, 
+  { x: '0', opacity: 1, ease:'back.out(1.7)', delay: 1.3 }
+);
+
+gsap.fromTo('.circle-large', 
+  { x: '-10rem', opacity: 0 }, 
+  { x: '0', opacity: 1, ease:'back.out(1.7)', delay: 1.9 }
+);
+
+gsap.fromTo('#intro-content', 
+  { x: '30rem', opacity: 0 }, 
+  { x: '0', opacity: 1, ease:'back.out(1.7)', delay: 2.5 }
+);
+
+/* event handlers */
+
+// Change colors and rotate arrow on hover
```

Comment:
Nit: [JsDoc for function comments](https://google.github.io/styleguide/jsguide.html#jsdoc-general-form) should be in the form ```/** Blah blah blah */```.